smart-contract/contracts/ElectionFactory.sol

```diff
@@ -54,11 +54,11 @@ contract ElectionFactory {
     }
 
     /// @dev Deploys a new election smart contract and stores the details.
-    function createElection(string memory _position, string[] memory _contestants) external onlyOwner {
+    function createElection(string memory _position, string[] memory _contestants, address[] memory _contestantsAddress) external onlyOwner returns(address) {
         uint256 count = electionCount;
         count++;
 
-        Election election = new Election(msg.sender, _position, _contestants.length, _contestants, count);
+        Election election = new Election(msg.sender, _position, _contestants.length, _contestants, _contestantsAddress);
```

Comment:
Please as discussed yesterday remove the contestant address? This looks like we are trying to dox our contestants.

Reply:
we need the address, we need something unique to recognize each contestant, Name can be the same but we can't have two address

Reply:
Collect their email please.